src/main/java/mate/academy/service/OrderServiceImpl.java

```diff
@@ -4,27 +4,30 @@
 import java.util.List;
 import mate.academy.model.Order;
 import mate.academy.model.Product;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class OrderServiceImpl implements OrderService {
+    private static final Logger logger = LogManager.getLogger(OrderServiceImpl.class);
+
     @Override
     public Order completeOrder(Long userId) {
-        // TODO: add log message about method completeOrder was called
+        logger.debug("Method: completeOrder. User id: {}", userId);
```

Comment:
Let's make our log messages informative. Include method parameters in the log message to make it clear which user's order is being completed.